pandas/core/tools/timedeltas.py

```diff
@@ -130,6 +131,9 @@ def to_timedelta(arg, unit='ns', box=True, errors='raise'):
 def _validate_timedelta_unit(arg):
     """ provide validation / translation for timedelta short units """
     try:
+        if arg == 'Y' or arg == 'y' or arg == 'M':
+            warnings.warn("Month and year units are deprecated.",
```

Comment:
In this warnings, explicitly call out the offset strings that are deprecated (i.e. `'Y'` and `'M'`)